ansible/mongodb-backup.yml

```diff
@@ -0,0 +1,9 @@
+- hosts: mongodb-backup
```

Comment:
@raghupathiguduri Lets have this as `"{{ hosts }}"` and pass this value from jenkins. We will have mongodb for nodebb and graylog, so we can have two jobs and pass the value from Jenkins job.